client/scss/components/forms/_nested-panel.scss

```diff
@@ -75,3 +75,7 @@ $panel-x-offset: calc($header-button-size-sm / 2 + $header-gap);
     display: none;
   }
 }
+
+.hidden {
```

Comment:
I'd like us to avoid using the added class, the current implementation uses the hidden attribute. Let's try to align with exactly how the current implementation is done (JavaScript wise). 

See TS code here. https://github.com/wagtail/wagtail/blob/main/client/src/includes/panels.ts

Reply:
are you suggesting to use attribute like aria-controls bacause if i remove hidden class from scss then it isnt working